sql/create_subcategory_table.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE subcategory (
+  subCategoryId INTEGER PRIMARY KEY AUTO_INCREMENT,
+  sub_category_name VARCHAR(255) NOT NULL,
+  category_id INT,
+  created_on TIMESTAMP,
+  last_updated_on TIMESTAMP,
+  archive BOOLEAN,
+  FOREIGN KEY (category_id) REFERENCES category (categoryId)
+);
```

Comment:
completed the code change of using Integer instead of int and use id instead of category_id and subCategory_id .
Updated and used name instead of category_name and subCategory_name